com.unity.render-pipelines.universal/Shaders/Lit.shader

```diff
@@ -91,6 +95,9 @@ Shader "Universal Render Pipeline/Lit"
             #pragma shader_feature _SMOOTHNESS_TEXTURE_ALBEDO_CHANNEL_A
             #pragma shader_feature _OCCLUSIONMAP
 
+            #pragma shader_feature _CLEARCOAT
+            #pragma shader_feature _CLEARCOATMAP
```

Comment:
Should we add a rule to the stripper to remove all shader variants  that define _CLEARCOATMAP but not _CLEARCOAT? This will save some variants.

Reply:
add rules to shaderpreprocessor.cs

Reply:
make _CLEARCOAT and _CLEARCOATMAP independent